src/openrct2/rct2/S6Importer.cpp

```diff
@@ -169,6 +169,11 @@ class S6Importer final : public IParkImporter
         return false;
     }
 
+    bool HasInvalidObjects() override
```

Comment:
Bugger, I knew I had it that way to start with for a reason! You're entirely right, I'll swap it back!